region-occurrences-highlighter.el

```diff
@@ -165,7 +165,9 @@
                                 (push buffer buffers)))))))
         (select-frame current-frame)
         buffers)
-    (list (current-buffer))))
+    (list (with-selected-window (or (minibuffer-selected-window)
+                                    (selected-window))
+            (current-buffer)))))
```

Comment:
The function `(minibuffer-selected-window)` will make to return the previously selected window. Else it will return nil and fall back to the currently selected window.